scripts/exregional_nexus_post_split.sh

```diff
@@ -7,7 +7,7 @@
 #
 #-----------------------------------------------------------------------
 #
-. $USHdir/source_util_funcs.sh
+. ${GLOBAL_VAR_DEFNS_FP}
```

Comment:
@bbakernoaa, can you remove this change? It was a recent update in the 'develop' branch.